third_party/packages/ansi_up/lib/ansi_up.dart

```diff
@@ -345,8 +347,8 @@ class StyledText {
       return '';
     }
     return <String>[
-      if (bgColor != null) 'background-color: ${_colorToCss(bgColor)}',
-      if (fgColor != null) 'color: ${_colorToCss(fgColor)}',
+      if (bgColor != null) 'background-color: ${_colorToCss(bgColor!)}',
```

Comment:
for these cases the recommended pattern is the slightly verbose
final bgColor = bgColor;
final fgColor = fgColor;
return [
  if (bgColor != null) 'background-color: ${_colorToCss(bgColor!)}',
...
];
Which avoids the extra `!`

Reply:
I don't see the difference between the code and what you wrote? bgColor is still being unwrapped?

Reply:
My fault. I left out the `!` that can be removed in the example.
@pq any plants to add a lint to suggest quick fixes that add local variables to avoid an extra `!`?
```dart
final bgColor = bgColor;
final fgColor = fgColor;
return [
  if (bgColor != null) 'background-color: ${_colorToCss(bgColor)}',
...
];
```